fe/fe-core/src/main/java/org/apache/doris/rewrite/ExprRewriter.java

```diff
@@ -36,11 +37,17 @@
 public class ExprRewriter {
     private int numChanges_ = 0;
     private final List<ExprRewriteRule> rules_;
+    private List<ExprRewriteRule> onceRules_ = Lists.newArrayList();
```

Comment:
Add comment to explain what is `onceRules`

Reply:
OK